### ISSUE TITLE
Structured logging to file, simple logging to console

### ISSUE THREAD

**Issue by pinx** [adams85/filelogger#20]
Could you provide a simple example of how to configure logging, such that the log files are filled with structured logging and the console with human readable logging?
Ideally, these settings would make it happen:
```json
    "Console": {
      "FormatterName": "simple",
      "FormatterOptions": {
        "SingleLine": true
      }
    },
    "File": {
      "FormatterName": "json",
      "FormatterOptions": {
        "SingleLine": true
      },
      "Files": [
        {
          "Path": "default-<date>.log"
        }
      ]
    }
```

**Comment by adams85** [comment]
The file logger library doesn't offer structured logging out-of-the-box currently. Although it's doable through some subclassing and overriding of methods, it requires pretty heavy customization.

I have plans to include structured logging capabilities (JSON, possibly) in the future but that will only happen in a new major version because it will need some breaking changes to the public APIs.

**Comment by adams85** [comment]
Implemented in v3.3.0.
